common/isolationgroup/defaultisolationgroupstate/state.go

```diff
@@ -74,14 +74,12 @@ func NewDefaultIsolationGroupStateWatcherWithConfigStoreClient(
 	}, nil
 }
 
-func (z *defaultIsolationGroupStateHandler) AvailableIsolationGroupsByDomainID(ctx context.Context, domainID string, tasklistName string, availablePollerIsolationGroups []string) (types.IsolationGroupConfiguration, error) {
+func (z *defaultIsolationGroupStateHandler) AvailableIsolationGroupsByDomainID(ctx context.Context, domainID string, _ string) (types.IsolationGroupConfiguration, error) {
```

Comment:
the behavior of this method changed
previously, it only returned undrained isolation groups but now it also returns drained groups
consider renaming it or doing it in a different way if you want to get drained groups

Reply:
Done.